app/routers/nutrition_log.py

```
@@ -6,15 +6,20 @@
 
 from __future__ import annotations
 
+from datetime import date, datetime
```

Flake8 (7.3.0)

[error] 'datetime.datetime' imported but unused

(F401)